src/SpacetimeDBClient.cs

```diff
@@ -1101,6 +875,33 @@ void IDbConnection.Unsubscribe(QueryId queryId)
         void IDbConnection.AddOnDisconnect(WebSocket.CloseEventHandler cb) => webSocket.OnClose += cb;
     }
 
+    internal struct ParsedDatabaseUpdate
+    {
+        // Map: table handles -> (primary key -> IStructuralReadWrite).
+        // If a particular table has no primary key, the "primary key" is just the row itself.
+        // This is valid because any [SpacetimeDB.Type] automatically has a correct Equals and HashSet implementation.
+        public Dictionary<IRemoteTableHandle, IParsedTableUpdate> Updates;
+
+        // Can't override the default constructor. Make sure you use this one!
+        public static ParsedDatabaseUpdate New()
+        {
+            ParsedDatabaseUpdate result;
+            result.Updates = new();
+            return result;
+        }
+
+        public IParsedTableUpdate DeltaForTable(IRemoteTableHandle table)
```

Comment:
Change name to `UpdateForTable`, comment here explaining that this returns the TableUpdate for a particular table, allocating one if none is present.